src/database.ts

```diff
@@ -1,24 +1,60 @@
-import { PrismaClient } from '@prisma/client';
+import { PrismaClient, Prisma } from '@prisma/client';
+import { Level as PinoLevel } from 'pino';
+import { getChildLogger } from './logger';
 import { getUserCrud } from './modules';
 
-let prisma: PrismaClient;
+const dbLogger = getChildLogger({ msgPrefix: 'DATABASE' });
 
-export const getPrisma = () => {
-  if (!prisma) {
-    prisma = new PrismaClient({
-      datasources: {
-        db: {
-          url: 'file:./db.sqlite',
-        },
-      },
-    });
-  }
-  const xprisma = prisma.$extends({
-    model: {
-      user: {
-        ...getUserCrud(prisma),
-      },
+const LOGGING_EVENTS: { level: Prisma.LogLevel; emit: 'event' }[] = [
+  {
+    emit: 'event',
```

Comment:
https://www.prisma.io/docs/concepts/components/prisma-client/working-with-prismaclient/logging#log-to-stdout

this is done as in docs, 

this can be `emit` or `stdout`, but since we want to recapture the logging events, it's always `emit`.

I find this approach to be most simple and clear for understanding due to explicit-ness.

Added a comment with ref to documentation on top of this `const`